### PR TITLE
adapt behavior for stdcm

### PR DESCRIPTION
**Pull request by alexandredamiron**

### CODE REVIEW

**Review by eckter — DISMISSED** [review]

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js [hunk 1]
```diff
@@ -11,11 +11,12 @@ import { trainscheduleURI } from 'applications/osrd/components/Simulation/consts
...
 import { TYPES_UNITS } from './consts';
 
+import { ALLOWANCE_UNITS_KEYS } from './consts';
+
 export default function StandardAllowanceDefault(props) {
   const {
     distributionsTypes,
-    allowanceTypes,
-    getAllowances,
+    ggetAllowances,
```
Comment (anisometropie) [inline front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: typo

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/applications/stdcm/formatStcmConf.ts [hunk 2]
```diff
@@ -2,12 +2,26 @@ import { Dispatch } from 'redux';
...
 
 import { STDCM_MODES, OsrdConfState } from 'applications/osrd/consts';
+import { TYPES_UNITS } from 'applications/osrd/components/Simulation/Allowances/consts';
 import { time2sec } from 'utils/timeManipulation';
 import { makeEnumBooleans } from 'utils/constants';
 
 import { ActionFailure } from 'reducers/main';
 import { ThunkAction } from 'types';
 
+type standardStdcmAllowanceForApi = {
```
Comment (anisometropie) [inline front/src/applications/stdcm/formatStcmConf.ts:12]: To be replaced when the openapi types are updated 
Screenshot_20230113_114148

File: front/src/applications/osrd/consts.ts [hunk 3]
```diff
@@ -141,4 +141,5 @@ export interface OsrdConfState {
   featureInfoClick: { displayPopup: boolean; feature?: Feature; coordinates?: number[] };
   gridMarginBefore: number;
   gridMarginAfter: number;
+  standardStdcmAllowance: any; // We wait for auto generated types
```
Comment (anisometropie) [inline front/src/applications/osrd/consts.ts:144]: To be replaced when the openapi types are updated

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js (hunk 1 above)
Comment (alexandredamiron) [reply front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: Done

File: front/src/applications/stdcm/formatStcmConf.ts (hunk 2 above)
Comment (alexandredamiron) [reply front/src/applications/stdcm/formatStcmConf.ts:12]: Agree

File: front/src/applications/osrd/consts.ts (hunk 3 above)
Comment (alexandredamiron) [reply front/src/applications/osrd/consts.ts:144]: Agree comment for that

**Review by anisometropie — APPROVED** [review]

**Pull request merged**